performance/Makefile

```diff
@@ -0,0 +1,56 @@
+# Docker image
+IMAGE_REPO ?= determinedai/perf-test
+IMAGE_TAG ?= $(USER)
+
+# Perf testing cluster
+CLUSTER_ID ?= $(USER)-perf
+KEYPAIR_ID ?= $(USER)
+VERSION ?= $(shell det -v | cut -d' ' -f2)
+
+# Perf test options
+DET_ADMIN_USERNAME ?= admin
+DET_ADMIN_PASSWORD ?= ""
+
+IMAGE = $(IMAGE_REPO):$(IMAGE_TAG)
+DET_URL ?= $(shell cat .det.url || echo UNSET)
+
+build: Dockerfile src/* package-lock.json package.json tsconfig.json webpack.config.js
```

Comment:
should the default target be something else?

Reply:
Is there a specific target you feel makes most sense?